db/migrate/20170307215002_rename_container_label_tag_mapping.rb

```diff
@@ -0,0 +1,36 @@
+class RenameContainerLabelTagMapping < ActiveRecord::Migration[5.0]
+  class MappableEntity < ActiveRecord::Base; end
+  class ProviderLabelTagMapping < ActiveRecord::Base; end
+
+  def change
+    create_table :mappable_entities do |t|
+      t.string :name,     :comment => "The name of the mappable resource type."
+      t.string :provider, :comment => "The provider name associated with the resource type."
+    end
+
+    # Add the current default list of mappable entities currently defined by
+    # the ContainerLabelTagMapping::MAPPABLE_ENTITIES array.
+    [
+      'ContainerProject',
+      'ContainerRoute',
+      'ContainerNode',
+      'ContainerReplicator',
+      'ContainerService',
+      'ContainerGroup',
+      'ContainerBuild',
+    ].each do |entity|
+      MappableEntity.create!(:name => entity, :provider => 'kubernetes')
+    end
+
+    MappableEntity.create!(:name => 'VmOrTemplate', :provider => 'amazon')
```

Comment:
I went back and forth on this, but I think that `:provider` should be the fully qualified name of the manager class.  

What occurred to me is that when the UI attempts to show the name of the provider, it's going to need a way to map to the [provider description](https://github.com/ManageIQ/manageiq-providers-amazon/blob/4eceeedda57509d5529466c4770c7db8afc48b9b/app/models/manageiq/providers/amazon/cloud_manager.rb#L97), and there's no way that I know of to do that with just `'amazon'`.  However, if the full classname was stored here, then the UI could just `contantize` the `provider` and call `provider.description`.